TCP raspberry/receiver.py

```diff
@@ -0,0 +1,236 @@
+import socket
+import select
+import os
+import traceback
+import picamera
+import time
+import threading
+import __future__
+import io
+
+# This import will throw a syntax error if the module is not installed.
+import serial
+
+# The encoding and decoding procedure to follow.
+coding = 'utf-8' # 'ISO8859-1'
```

Comment:
Because we changed encoding method several times. And remembering ISO8859-1 is hard.